app/scripts/common/components/oo-savebutton.directive.js

```diff
@@ -56,6 +56,11 @@ angular.module('openolitor-core').directive('ooSaveButton', ['msgBus',
             DataUtil.update(msg.data, $scope.model);
             $scope.model.actionInProgress = undefined;
             if ($scope.onCreated) {
+              if ($scope.form) {
+                if ($scope.form.destroyConfirmOnDirty) {
```

Comment:
You might want to check if "$scope.form.destroyConfirmOnDirty" is a function adding this: "&& typeof $scope.slideOutFunction === 'function'"